example/ios/KlaviyoReactNativeSdkExampleExtension/NotificationService.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NotificationService.swift
+//  KlaviyoReactNativeSdkExampleExtension
+//
+//  Created by Ajay Subramanya on 4/9/24.
+//
+
+import KlaviyoSwiftExtension
+import UIKit
+import UserNotifications
+
+// MARK: notification service extension implementation.
+
+/// When push payload is marked as there being mutable-content this service
```

Comment:
total nit but: `When a push payload` sounds a little better

Reply:
Or maybe even `When a push payload is marked as having mutable-content this service`...

Reply:
I removed this as this is sample code from iOS repo and probably not required here. Will fix it there in a different PR.